Firestore/core/src/firebase/firestore/local/leveldb_migrations.cc

```diff
@@ -244,10 +244,11 @@ void EnsureSentinelRows(leveldb::DB* db) {
 }  // namespace
 
 LevelDbMigrations::SchemaVersion LevelDbMigrations::ReadSchemaVersion(
-    LevelDbTransaction* transaction) {
+    leveldb::DB* db) {
+  LevelDbTransaction transaction(db, "ReadSchemaVersion");
```

Comment:
Other transaction descriptions use an english phrase, so please preserve it as "Read schema version".

Reply:
Done.